src/main/resources/init_db.sql

```diff
@@ -18,3 +18,18 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+
+CREATE TABLE `taxy_service.cars`(
```

Comment:
```suggestion
CREATE TABLE `cars`(
```

Change it in all sql query